int64 enc_L
int64 enc_R
float64 d_A
float64 d_theta
float64 abs_distance
float64 abs_angle